src/DataStructures/Tensor/EagerMath/Magnitude.hpp

```diff
@@ -48,11 +50,56 @@ Scalar<DataType> magnitude(
 /// \details
 /// Computes the square root of the absolute value of the scalar.
 template <typename DataType>
-Scalar<DataType> sqrt_magnitude(
-    const Scalar<DataType>& input) noexcept {
+Scalar<DataType> sqrt_magnitude(const Scalar<DataType>& input) noexcept {
   return Scalar<DataType>{sqrt(abs(get(input)))};
 }
 
+/// \ingroup TensorGroup
+/// \brief Compute point-wise $\\ell^2$ (Frobenius) norm of a tensor
```

Comment:
Would these functions be better placed in a `Norms.hpp` file? Would do you think @prayush and @fmahebert ?

Reply:
moved code to `Norms.hpp`!